### PR TITLE
doc: Add section on configuration to 'Getting started'

### PR DESCRIPTION
**Pull request by leandrolanzieri**
### Contribution description
This PR adds a small section on application configuration, which references the 'Compile-time configurations' group and the Kconfig section and user's guide. It also moves Kconfig section closer to build system in the doc menu.

### Testing procedure
- Build the docs and read

### Issues/PRs references
None

### CODE REVIEW

File: doc/doxygen/src/getting-started.md [hunk 1]
```diff
@@ -140,3 +140,15 @@ make -C examples/gnrc_networking/ term \
     TERMPROG=gtkterm \
     TERMFLAGS="-s 115200 -p /dev/ttyACM0 -e"
 ~~~~~~~~
+
+Configuring an application                         {#configuring-an-application}
+--------------------------
+Many modules in RIOT offer configuration options in compile-time.
```
Comment (waehlisch) [inline doc/doxygen/src/getting-started.md]: configuration options that will be considered during compile-time.

Comment (leandrolanzieri) [reply doc/doxygen/src/getting-started.md]: Changed.

**Comment by leandrolanzieri** [comment]
I addressed the comment and squashed directly.

**Comment by waehlisch** [comment]
ACK. When the CI gives green lights, feel free to merge.

**Pull request merged**